oops/backplane/pixel.py

```diff
@@ -0,0 +1,65 @@
+################################################################################
+# oops/backplanes/pixel.py: pixel coordinate backplanes
+################################################################################
+
+from oops.constants import C
+from oops.backplane import Backplane
+from oops.body import Body
+
+#===============================================================================
+def body_diameter_in_pixels(self, event_key, radius=0):
+    """Gridless approximate apparent diameter of the body in pixels.
+
+    Input:
+        event_key       key defining the event on the body's path.
+        radius          If nonzero, override the radius of the body referred to
+                        in the event key.
+    """
+    gridless_key = Backplane.gridless_event_key(event_key)
+
+
+    key = ('body_diameter_in_pixels', gridless_key, radius)
+    if key in self.backplanes:
+        return self.get_backplane(key)
+
+    # compute apparent distance
+    event = self.get_surface_event(gridless_key, arrivals=True)
+    distance = event._dep_lt_*C
+
+    # compute apparent enclosing radius
+    if radius==0:
+        (body, mod) = Backplane.get_body_and_modifier(gridless_key[1])
+        radius = body.radius
+    radius = radius/distance / self.obs.fov.uv_scale.values[0] ### only good for snapshots
+
+    return 2*self.register_backplane(key, radius)
+
+#===============================================================================
+def center_coordinate(self, event_key, axis="x"):
+    """Gridless v coordinate of the center of the disk.
```

Comment:
I think you mean "v" to be "x or y" in the docstring.

Reply:
Actually  v is just spurious.  I don't think the "x or y" is needed.